src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -83,26 +83,26 @@ class DatePicker extends React.Component {
   }
 
   formatTextfieldDate() {
-    const { open } = this.state;
+    // const { open } = this.state; // Uncomment when we enhance for typing in datefield
```

Comment:
I am aware there are comments here, but I think they are important and relevant for when we go back to enhance this functionality.